core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/database/ExternalDatabaseToDatabaseResponseConverter.java

```diff
@@ -6,11 +6,14 @@
 import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.response.database.DatabaseResponse;
 
 @Component
-public class DatabaseAvailabilityTypeToDatabaseResponseConverter {
+public class ExternalDatabaseToDatabaseResponseConverter {
 
-    public DatabaseResponse convert(DatabaseAvailabilityType source) {
+    public DatabaseResponse convert(DatabaseAvailabilityType availabilityType, String externalDatabaseEngineVersion) {
         DatabaseResponse response = new DatabaseResponse();
-        response.setAvailabilityType(source);
+        if (availabilityType != null) {
+            response.setAvailabilityType(availabilityType);
+        }
+        response.setDatabaseEngineVersion(externalDatabaseEngineVersion);
         return response;
```

Comment:
I suppose we should send back the result as null when both input fields are null